src/playlist.tsx

```diff
@@ -192,6 +201,12 @@ export class Playlist extends KalturaPlayer.core.BasePlugin {
     });
   };
 
+  private _handleError = () => {
+    if (this._player.playlist?.items?.length > 1) {
+      this.player.playlist.playNext();
```

Comment:
what happens if we have an error in the last entry ?

Reply:
jump to first one, the feature handles by playlist logic